### PR TITLE
Switch to config directory for rules

### PR DESCRIPTION
**Pull request by achow**
@jeeyoungk

### CODE REVIEW

File: internal/api.go [hunk 1]
```diff
@@ -43,6 +37,38 @@ func NewAPI(config api.Config) (api.API, error) {
...
+	ruleSet := RuleSet{
+		rules: []Rule{},
+	}
+
+	filenames, err := filepath.Glob(filepath.Join(conversionRulesPath, "*.yaml"))
+	if err != nil {
+		return RuleSet{}, err
+	}
+
+	for _, filename := range filenames {
```
Comment (jeeyoungk) [inline internal/api.go]: can we sort the filenames so that we can have a deterministic loading?

File: internal/api.go [hunk 2]
```diff
@@ -43,6 +37,38 @@ func NewAPI(config api.Config) (api.API, error) {
...
+		file, err := os.Open(filename)
```
Comment (jeeyoungk) [inline internal/api.go]: do we need to close files? (i've never done that in go so i have no clue whether that's needed or not).

Comment (Nathan-Fenner) [inline internal/api.go]: The Go idiom is to have `defer file.Close();` immediately after opening a file (unless it needs to stay open after you return for some reason).

Comment (Nathan-Fenner) [inline internal/api.go]: It's idiomatic to check for an error opening the file, and then defer closing the file.

For reference: https://gobyexample.com/writing-files

**Comment by jeeyoungk** [comment]
LGTM

**Pull request merged**